### PR TITLE
Convert Shader to template literal

### PR DESCRIPTION
**Pull request by RobertGemmaJr**
Converts the vertexShader and fragmentShader in `Shader.js` to template strings. I fixed the tabbing and cleaned up the code.

Would still love to move them into their own files but reading the text has been oddly difficult inside the aframe files. Looks like aframe has its own [shader component](https://aframe.io/docs/1.3.0/components/material.html#schema) so I might move in that direction when I start physically blending them together. This PR at least organizes the code for little effor

### CODE REVIEW

File: src/Aframe/fragment-shader.frag [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+uniform float dim;
+uniform sampler2D u_data; //volume dataset
+uniform mat4 clipPlane; 
+uniform bool clipping;
+uniform float threshold;
+uniform float multiplier;
+uniform vec3 step_size; //ray step size
+uniform int channel;
+uniform sampler2D u_lut; //transfer function
+uniform bool useLut;
```
Comment (mcmcgrath13) [inline src/Aframe/fragment-shader.frag:13]: 🌈

File: src/Aframe/model.js [hunk 2]
```diff
@@ -176,8 +178,24 @@ AFRAME.registerComponent("model", {
         const zScale = volumeScale[0] / volumeScale[2];
 
         // Set material properties
-        const shader = THREE.ShaderLib["ModelShader"];
-        const uniforms = THREE.UniformsUtils.clone(shader.uniforms);
+        // const shader = THREE.ShaderLib["ModelShader"];
+        // const uniforms = THREE.UniformsUtils.clone(shader.uniforms);
```
Comment (mcmcgrath13) [inline src/Aframe/model.js]:
```suggestion
```

File: src/Aframe/model.js [hunk 3]
```diff
@@ -198,9 +216,9 @@ AFRAME.registerComponent("model", {
 
         const material = new THREE.ShaderMaterial({
           uniforms: uniforms,
+          vertexShader: vertexShader,
+          fragmentShader: fragmentShader,
```
Comment (mcmcgrath13) [inline src/Aframe/model.js:218]:
```suggestion
          uniforms,
          vertexShader,
          fragmentShader,
```

File: src/Aframe/fragment-shader.frag [hunk 4]
```diff
@@ -0,0 +1,165 @@
+#define FILTER_LIN 1
+
+uniform float slice;
+uniform float dim;
```
Comment (mcmcgrath13) [inline src/Aframe/fragment-shader.frag:4]: out of curiosity, why are the extensions on these files `.frag` and `.vert` and not the `.gsls` or `.c` (or whatever the actual language is?

Comment (RobertGemmaJr) [reply src/Aframe/fragment-shader.frag:4]: I asked Camillo and he said it was just convention. The official language is the OpenGl Shader Language (`.glsl`), I gather `.frag` and `.vert` are just used to specify whether the code defines a fragment or vertex shader. 

cc @kmilo9999

**Review by mcmcgrath13 — APPROVED** [review]

**Pull request merged**